python/ql/src/Expressions/CallArgs.qll

```diff
@@ -71,36 +73,38 @@ private int positional_arg_count_for_call(Call call, Value callable) {
     )
 }
 
+/** Gets the number of arguments in `call`. */
 int arg_count_objectapi(Call call) {
     result = count(call.getAnArg()) + varargs_length_objectapi(call) + count(call.getAKeyword())
 }
 
+/** Gets the number of arguments in `call`. */
```

Comment:
This description is a duplicate of the one above - do they really produce exactly the same results? 

This is true of all the changes in this library. I guess it must be intended but to the uninformed, it looks strange to have two ways of doing the same thing.

Reply:
They don't produce _exactly_ the same results, but mostly yes. The reason we have this duplication is that one copy uses the old "Object API" (hence the suffix), and the other uses the Value API (which is what we are switching over to). Once all dependence on `arg_count_objectapi` and its brethren has been eliminated, we can mark them as deprecated.

(Also, I expect that this file is only really used internally, by the queries that check for invalid function invocations.)

Reply:
Thanks 👍